content/projects/minesweeper.md

```diff
@@ -1,5 +1,5 @@
 ---
-featured: true
+featured: false
```

Comment:
Please revert the `featured` flags on the 3 projects back to true, since the index page featuredProjects section is filtering by that flag and now might be empty.

Reply:
The rest looks good to me. Well done.

Reply:
Changes Pushed @Knochenmark